apps/api/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
```

Comment:
Did you check the docker running after making these change?

Reply:
Building and running the docker did not give any errors 
![image](https://github.com/OsmosysSoftware/osmo-x/assets/108812833/4f2d818f-501f-44c3-95a8-766a9e45e364)